app/models/service/aggregation.rb

```diff
@@ -315,7 +315,7 @@ def aggregate_all_vm_disk_space_used
   def aggregate_all_vm_memory_on_disk
     if has_attribute?("aggregate_all_vm_memory_on_disk")
       # Avoids (poorly) "ERROR:  integer out of range" in postgres
-      self["aggregate_all_vm_memory_on_disk"] * 1.megabyte
+      self["aggregate_all_vm_memory_on_disk"].try(:*, 1.megabyte)
```

Comment:
I'm not exactly sure what this method represents, but is it supposed to return an integer or nil?  I personally would be tempted to use `.to_i`, but I don't know what memory_on_disk is.